uiowa/uiowa
# Education - Site Split

establish site split to combine AoS, profiles permissions

# Story

As a webmaster on Education, I should be able to create AoS content and configure the Directory Profiles settings.

# To Test

- sync education, enable site split and run `drush @education.local config:import --source ../config/sites/education.uiowa.edu --partial`

- Confirm that as a webmaster you can create Areas of Study and access the Directory Profiles configuration.
- Ensure I am following the site split best practices which _just_ changed.
